examples/with-react-intl/components/PageWithIntl.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react'
+import {IntlProvider, addLocaleData, injectIntl} from 'react-intl'
+
+// Register React Intl's locale data for the user's locale in the browser. This
+// locale data was added to the page by `pages/_document.js`. This only happens
+// once, on initial page load in the browser.
+if (typeof window !== 'undefined' && window.ReactIntlLocaleData) {
```

Comment:
From the locale data modules; e.g. https://unpkg.com/react-intl@2.2.3/locale-data/en.js

The server selects one React Intl's locale data scripts and inlines it on the page. Since the locale data is packaged as a UMD module, in the browser it creates the `window.ReactIntlLocaleData` global.

More details here: https://github.com/yahoo/react-intl/wiki#locale-data-in-browsers